app/services/geek_jokes.js

```diff
@@ -0,0 +1,19 @@
+// const logger = require('pino')();
```

Comment:
As a suggestion, I see that you named the file correctly when placing it in plural but in the card they talk about creating a service called weets, so when you are in the project it is ideal to follow the instructions of the cards.